openapi/v2/core-metadata.yaml

```diff
@@ -109,8 +109,6 @@ components:
     Device:
       type: object
       properties:
-        apiVersion:
```

Comment:
Still needed. We added apiVersion to all objects that can be part of a response.

Reply:
Why it needs to use two apiVersion in the response?
```
{
  "apiVersion": "string",
  "requestId": "e6e8a2f4-eb14-4649-9e2b-175247911369",
  ...
  "profile": {
    "apiVersion": "string",
    ...
  }
}
```

BaseResponse already has an apiVersion.
https://github.com/edgexfoundry/go-mod-core-contracts/blob/master/v2/dtos/common/base.go#L20

Reply:
Got it, added it back and create PR to add Versionable to DTO https://github.com/edgexfoundry/go-mod-core-contracts/pull/267